src/context-providers/display/display-provider.tsx

```diff
@@ -8,6 +8,7 @@ import { IDisplayState, TDisplayAction, TDisplayContext } from "./types";
 // =============================================================================
 const DEFAULT_DISPLAY_VALUES: IDisplayState = {
     toasts: [],
+    modalState: [],
```

Comment:
since we named toasts instead of toastState, let's keep to this naming convention
```suggestion
    modals: [],
```

Reply:
fixed in the last push